src/resources/Enums.ts

```diff
@@ -897,13 +897,11 @@ export enum SearchHubRawMetrics {
 
 export enum SearchHubLicenseMetrics {
     normalQueries = 'normalQueries',
-    recommendationsQueries = 'recommendationsQueries',
     commerceProductListingQueries = 'commerceProductListingQueries',
-    agents = 'agents',
```

Comment:
:suspect:

Reply:
haha you're too fast :P. I just pushed it